GCRCatalogs/catalog_configs/cosmoDC2_v1.1.yaml

```diff
@@ -0,0 +1,26 @@
+subclass_name: cosmodc2.CosmoDC2GalaxyCatalog
+
+catalog_root_dir: /global/projecta/projectdirs/lsst/groups/CS/cosmoDC2/cosmoDC2_v1.1.0
+catalog_filename_template: z_{}_{}.step_all.healpix_{}.hdf5
+
+cosmology:
+    H0: 71.0
+    Om0: 0.2648
+    Ob0: 0.0448
+    sigma8: 0.8
+    n_s: 0.963
+
+lightcone: true
+
+version: "1.1.0"
+
+check_md5: false
+
+check_size: false
+
+creators: ['Andrew Hearin', 'Danila Korytov', 'Eve Kovacs', 'Esteban Rangel', 'Patricia Larsen', 'Joe Hollowed', 'Andrew Benson', 'Katrin Heitmann']
+
+description: |
+    This is the extra-galactic catalog for the LSST-DESC data challenge DC2.
+
+included_by_default: true
```

Comment:
Let's just put the most recent one in the default list.
```suggestion
```

Also, should we rename this to `v1.1.0`?